src/profiles.js

```diff
@@ -37,4 +47,17 @@ const mergeProfiles = (profiles1, profiles2) => {
 
 const PROFILES = mergeProfiles(BUILTIN_PROFILES, CUSTOM_PROFILES);
 
+for (const cmd of Object.keys(PROFILES)) {
```

Comment:
Could we write out a simple example of `PROFILES` after this step?

Reply:
It kind of does what you'd expect - it copies the profile to each name listed in `aliases`, with an extra property `isAlias: true` to be able to filter it out in `sop list-profiles`. But yeah, I can write a comment to describe that